setup_project_with_framework.sh

```diff
@@ -0,0 +1,47 @@
+#! /bin/sh
+
+startdir=$(pwd)
+
+cmssw_arch="slc6_amd64_gcc530" ## should be in sync with CMSSW.arch
+cmssw_version="CMSSW_8_0_30"   ## should be in sync with CMSSW.release
+
+scramv1 -a "${cmssw_arch}" project CMSSW "${cmssw_version}"
+cd "${cmssw_version}"
+cmsenv
+
+git clone -o upstream https://github.com/cp3-llbb/Framework
+fwkdir_tmp="${CMSSW_BASE}/Framework"
+
+upstream_cmssw_arch=$(cat "Framework/CMSSW.arch")
+if [ "${upstream_cmssw_arch}" != "${cmssw_arch}" ]; then
+  echo "Platform mismatch with CMSSW.arch: ${upstream_cmssw_arch}, while set up with ${cmssw_arch}"
+  exit 1
+fi
+upstream_cmssw_version=$(cat "Framework/CMSSW.release")
+if [ "${upstream_cmssw_version}" != "${cmssw_version}" ]; then
+  echo "CMSSW version mismatch with CMSSW.release: ${upstream_cmssw_version}, while set up with ${cmssw_version}"
+  exit 1
+fi
+
+## apply recipes
+cd "${CMSSW_BASE}/src"
+. "${fwkdir_tmp}/bootstrap_jenkins.sh"
+## move Framework to the right place
+llbbdir="${CMSSW_BASE}/src/cp3_llbb"
+mkdir -p "${llbbdir}"
+mv "${fwkdir_tmp}" "${llbbdir}/"
```

Comment:
I'm not sure I like moving repos after checking out... basically the only thing that prevents doing the clone directly in the right place is the `git cms-init` if I am not mistaken, maybe adding one in this file should not hurt calling the `bootstrap_jenkins.sh` after ?

Reply:
it's more tricky: `git cms-init` needs the `${CMSSW_BASE}/src` directory to be empty, so as long as one is there in `bootstrap_jenkins.sh` you cannot have Framework in the right place until after that is called.

Reply:
moving a git repository should be fine, see https://stackoverflow.com/questions/17568543/git-add-doesnt-work/17747571#17747571 for details (we don't even use submodules)

Reply:
ok, fine then :) thanks !